spec/std/concurrent/select_spec.cr

```diff
@@ -107,6 +144,56 @@ describe "select" do
     x.should eq 1
   end
 
+  it "select fiber has one chance to be enqueued into scheduler (1)" do
+    ch1 = Channel(Int32).new
+    ch2 = Channel(Int32).new
+    ch3 = Channel(Int32).new
+
+    spawn do
+      select
+      when x = ch1.receive
+      when x = ch2.receive
+      end
+
+      x.should eq(1)
```

Comment:
assertions, seems not work correct when inside spawn, it just crash fiber when failed.

Reply:
Yup, the others use helper functions that catch spec exceptions.